util/county_to_dcid.py

```diff
@@ -155,7 +178,8 @@
         "Pinal County": "geoId/04021",
         "Santa Cruz County": "geoId/04023",
         "Yavapai County": "geoId/04025",
-        "Yuma County": "geoId/04027"
+        "Yuma County": "geoId/04027",
+        "Arizona": "geoId/04"
```

Comment:
Any reason to have state codes in the county map?

Reply:
Removed State Codes